packages/server/graphql/public/types/MeetingSeries.ts

```diff
@@ -18,7 +18,7 @@ const MeetingSeries: MeetingSeriesResolvers = {
       .orderBy((doc) => (doc('endedAt') ? 0 : 1), r.desc('createdAt'))
       .limit(1)
       .run()
-    return meetings[0]
+    return meetings[0]!
```

Comment:
-1 should we change the GQL Schema to make `mostRecentMeeting` nullable? It's possible they've never run one before

Reply:
I'm not too deep into meeting series. I think there cannot be a meeting series without at least one meeting.

Reply:
oh! fair point, LGTM